.github/ISSUE TEMPLATES/config.yml

```diff
@@ -0,0 +1,13 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Contributing guidelines
+    url: https://github.com/cardano-base/blob/master/CONTRIBUTING.md
+    about: Some rules & processes we honor.
+
+  - name: Feature ideas
+    url: https://github.com/cardano-base/discussions/categories/ideas
+    about: Maybe someone else had the same or a similar idea already?
+
```

Comment:
There is no need for discussions in cardano-base. Opening an issue for any reason is totally fine.
```suggestion
```